fedimint-client/src/db.rs

```diff
@@ -324,24 +324,14 @@ pub async fn apply_migrations_client(
     module_instance_id: ModuleInstanceId,
     decoders: ModuleDecoderRegistry,
 ) -> Result<(), anyhow::Error> {
-    // TODO(support:v0.3):
```

Comment:
On further inspection I realized this comment isn't exactly right, since we're relying on this function to create the database version for newly created databases too.

That's what prompted the change from `migrate_database_version` to `create_database_version`. So we can't remove this after 0.3, but I think that's ok since for newly created database, all this will do is write the database version to disk.